src/cgroup.rs

```diff
@@ -59,19 +59,30 @@ impl Default for Cgroup {
 }
 
 impl Cgroup {
+    pub fn v2(&self) -> bool {
+        self.hier.v2()
+    }
+
     /// Create this control group.
     fn create(&self) {
         if self.hier.v2() {
-            let _ret = create_v2_cgroup(self.hier.root(), &self.path);
+            let _ret = create_v2_cgroup(self.hier.root(), &self.path, None);
```

Comment:
`create_v2_cgroup` may be failed but why does the error is dropped?

Reply:
Sorry, fixed.